[quant] Refactoring the mappings files

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#44847 [quant] Refactoring the mappings files**

Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

Commit:
[quant] Refactoring the mappings files

[ghstack-poisoned]

jerryzh168 reviewed (approved):
I thought I already removed default_mappings.py..

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]

Commit:
Update on "[quant] Refactoring the mappings files"


Differential Revision: [D23747007](https://our.internmc.facebook.com/intern/diff/D23747007)

[ghstack-poisoned]
